check-lint

```diff
@@ -1,9 +1,26 @@
 #!/bin/sh -e
 
-TOOL_CMD=pylint
-PATH=$(basename $0)/utils:$PATH
+CONFIG_FILE="../.avocado_static_checks.conf"
+DEFAULT_CONFIG_PATH="default_configs/pylintrc"
 CONFIG_PATH="$(dirname $0)/default_configs/pylintrc"
+
+get_config() {
+    local file_path="$1"
+    if [ -f "$CONFIG_FILE" ]; then
+        while IFS=':' read -r pattern config; do
+            if [[ "$file_path" == $pattern* ]]; then
```

Comment:
Can you please add example how the config file should look like. Because from this it seems to me that the configuration can differ for each file in the SUT. To be hones, I am a little bit confused here.